create-doks-with-terraform-flux/variables.tf

```diff
@@ -0,0 +1,76 @@
+# DOKS
+variable "do_token" {
+  type = string
+  description = "DigitalOcean API token"
+  sensitive = true
+}
+
+variable "doks_cluster_name" {
+  type = string
+  default = "doks-fluxcd-cluster"
+  description = "DOKS cluster name"
+}
+
+variable "doks_cluster_region" {
+  type = string
+  default = "lon1"
+  description = "DOKS cluster region"
+}
+
+variable "doks_cluster_version" {
+  type = string
+  default = "1.21.2-do.2" # Grab the latest version slug from `doctl kubernetes options versions`
+  description = "Kubernetes version provided by DOKS"
+}
+
+variable "doks_cluster_pool_size" {
+  type = string
+  default = "s-2vcpu-4gb"
+  description = "DOKS cluster node pool size"
+}
+
+variable "doks_cluster_pool_node_count" {
```

Comment:
I'm getting this error when I run `terraform apply`: 

```

│ Error: Invalid default value for variable
│
│   on variables.tf line 34, in variable "doks_cluster_pool_node_count":
│   34:   default = 2   # need 2 nodes at least
│
│ This default value is not compatible with the variable's type constraint: map of any single type required.
```

Reply:
This change seemed to work: 
```variable "doks_cluster_pool_node_count" {
  type = number
  default = 2   # need 2 nodes at least
  description = "DOKS cluster node count inside the pool"
}```

Reply:
Thanks for testing. Seems that I forgot to change the variable type after setting the cluster to a "fixed" size. Should be fixed in the latest commit.